src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/RoboticsDataMapper.java

```diff
@@ -217,7 +228,33 @@ private Solicitor buildApplicantSolicitor(CaseData caseData, String id) {
                 RESPONDENT_ID,
                 RESPONDENT_SOLICITOR_ID
             )
-        );
+        ));
+
+        if (caseData.getApplicant2() != null) {
+            respondentParties.add(buildLitigiousParty(
+                caseData.getApplicant2(),
+                caseData.getApplicant2LitigationFriend(),
+                caseData.getApplicant2OrganisationPolicy(),
+                "Claimant",
+                APPLICANT2_ID,
+                APPLICANT_SOLICITOR_ID
```

Comment:
this has to be applicant 2 solicitor

Reply:
Also need to consider the scenario here if there are 2 applicants and both solicitors are same then use the same solicitor id

Reply:
Also need to consider the scenario here if there are 2 applicants and both have same solicitors, same logic as used for respondent:
`respondent2SolicitorId = caseData.getRespondent2SameLegalRepresentative() == Y ? RESPONDENT_SOLICITOR_ID : RESPONDENT2_SOLICITOR_ID;`